docs/recipes/type-checking/use-types.ts

```diff
@@ -0,0 +1,22 @@
+import * as lhApi from 'lighthouse';
```

Comment:
can you add license and a short readme to the directory? It is supposed to be a recipe, not just a test :)